README.md

```diff
@@ -3,21 +3,20 @@ gist-doc
 
 This repository contains documentation of the [Semantic Arts gist minimalist upper ontology](https://github.com/semanticarts/gist/tree/master) in three formats: narrative, graphical, and Widoco-style.
 
-Current version of gist: 11.0
+Current version of gist: 12.0.0
```

Comment:
Are we going to refresh this only at major releases? And not have to update this line every time we publish a new version of gist? Then we should say 

Current major version of gist: 12

Or 

Current version of gist: 12.x.x ("Current version" not quite right here, I don't know how to refer to this).

Reply:
I would also name the eBook in some way that doesn't make it out of date with patch or minor versions.